src/surrogates/inc/InterpolationSurrogateBase.h

```diff
@@ -0,0 +1,69 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// QUESO - a library to support the Quantification of Uncertainty
+// for Estimation, Simulation and Optimization
+//
+// Copyright (C) 2008-2015 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef UQ_INTERPOLATION_SURROGATE_BASE_H
+#define UQ_INTERPOLATION_SURROGATE_BASE_H
+
+#include <queso/SurrogateBase.h>
+#include <queso/BoxSubset.h>
+#include <queso/VectorSpace.h>
+#include <queso/InterpolationSurrogateData.h>
+
+namespace QUESO
+{
+  //! Base class for interpolation-based surrogates
+  /*! This class is used for surrogoate approximations of a model using interpolation.
```

Comment:
Hmm. I wonder if maybe the base class should be renamed to reflect that this is designed for single quantity output (vs. multiple quantity)?